docs/section23/Rule23-14.md

```diff
@@ -0,0 +1,38 @@
+
+# Airside System - Rule 23-14 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** False  
+**Rule ID:** 23-14  
+**Rule Description:** If the baseline system does not comply with exceptions in Section 6.5.2.3 then only 25% of the system reheat energy shall be included in the baseline building performance  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.18 Dehumidification (Systems 3 through 8 and 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** HVAC System  
+
+**Applicability Checks:**  
+
+1. P-RMR is modeled with at least one air-side system that includes humidistatic controls.  
+
+**Function Calls:**  
+
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys
+
+
+**Applicability Checks:**  
+- get a list of the hvac systems and the zones they served in the P_RMR: `hvac_systems_and_zones_p = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(P_RMR)`
+- look through each hvac system in the proposed: `for hvac_p_id in hvac_systems_and_zones_p:`
+    - get the proposed hvac sytem: `hvac_p = get_object_by_id(hvac_p_id)`
+    - get the cooling system: `cooling_system_p = hvac_p.cooling_system`
+    - check to see if the cooling system has a humidification type other than NONE, if so, there are humidification controls = TRUE: `if cooling_system_p.humidification_type != NONE: `
+        - look through each zone served by the system: `for zone_id in hvac_systems_and_zones_p[hvac_p_id]["ZONE_LIST"]:`
+            - get the zone: `zone_p = get_object_by_id(zone_id)`
+            - if the zone_p.maximum_humidity_setpoint_schedule exists, the rule is applicable, return UNDETERMIEND: `if zone_p.maximum_humidity_setpoint_schedule: UNDETERMINED`
```

Comment:
In 23-13 you check for maximum and minimum, why just max here? 

See comment in 23-13 regarding conducting the check at the HVAC system level and including an output message for the reviewer.